Gemfile

```diff
@@ -2,19 +2,23 @@
 
 source 'https://rubygems.org'
 
-gem 'dry-validation'
-gem 'foreman'
-gem 'pg'
-gem 'puma'
-gem 'roda'
-gem 'sequel'
 gem 'figaro', '~>1.2'
+gem 'foreman', '~>0.0'
```

Comment:
Great question – the foreman gem is version is currently `0.87.2` (where 0 is major version, 87 is minor version, and 2 is patch version)

using `~>0.0` ensures that we do not upgrade beyond version `0.*` for now. The squiggly arrow `~>` means that it will only increase the smallest specified number, which is the minor version in `~>0.0`.